publisher/monitor/price/monitor-price.py

```diff
@@ -40,10 +37,7 @@ async def main():
 
     all_prices_valid = True
     for asset in assets:
-        if "pair" in asset:
-            key = currency_pair_to_key(*asset["pair"])
-        else:
-            key = asset["key"]
+        key = key_for_asset(key)
         felt_key = str_to_felt(key)
         if felt_key not in coingecko or asset["type"] != "SPOT":
             print(
```

Comment:
add sources to get_value call

Reply:
Nevermind, this is a nice default on the SDK (passes empty list if sources is None)